packages/core-flows/src/invite/steps/update-invites.ts

```diff
@@ -0,0 +1,50 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  IUserModuleService,
+  UpdateInviteDTO,
+  UpdateUserDTO,
+} from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+export const updateInvitesStepId = "update-invites-step"
+export const updateInvitesStep = createStep(
+  updateInvitesStepId,
+  async (input: UpdateInviteDTO[], { container }) => {
+    const service: IUserModuleService = container.resolve(
+      ModuleRegistrationName.USER
+    )
+
+    if (!input.length) {
+      return new StepResponse([], [])
+    }
+
+    const originalInvites = await service.listInvites({
+      id: input.map((u) => u.id),
+    })
+
+    if (!originalInvites?.length) {
+      return new StepResponse([], [])
+    }
+
+    const invites = await service.updateInvites(input)
+    return new StepResponse(invites, originalInvites)
+  },
+  async (originalInvites, { container }) => {
+    if (!originalInvites?.length) {
+      return
+    }
+
+    const service = container.resolve(ModuleRegistrationName.USER)
+
+    await service.updateInvites(
+      originalInvites.map((invite) => ({
+        id: invite.id,
```

Comment:
todo: ID should not be updatable

suggestion: I would probably also remove email from the updatable fields. In the case of admins writing the email wrong, they would revoke and create a new one. That way, we only have to handle email triggers on creation (and resending). Wdyt?
```suggestion
```

Reply:
And perhaps, also expires at 😊